src/bpftrace.cpp

```diff
@@ -1959,6 +1966,20 @@ int BPFtrace::resolve_uname(const std::string &name,
 #endif
 }
 
+std::string BPFtrace::resolve_mac_address(const uint8_t *mac_addr) const
+{
+  char addr[18];
+  sprintf(addr,
```

Comment:
```suggestion
  snprintf(addr, 18
```

Not needed but just to be safe anyway